### DISCUSSION TITLE
On mobile will be nice to swipe up and down to change month

### DISCUSSION THREAD

**Discussion by davidegolzi** [dmuy/duDatepicker#43] [Ideas]
should really great to have this functionality instead using buttons to change month like material concept

[https://material.io/components/date-pickers#mobile-pickers](https://material.io/components/date-pickers#mobile-pickers)

maybe going full screen also :)

**Comment by dmuy** [comment]
Hi @davidegolzi 

Thanks for this suggestion, I've pushed a basic implementation of swiping left/right to change month (on mobile) - [v2.0.4](https://github.com/dmuy/duDatepicker/releases/tag/v2.0.4)

As for going fullscreen, I'll think about it.

*Note: I did swipe left/right instead of up/down to go with arrow left/right buttons.

**Reply by davidegolzi** [reply]
hi @dmuy thanks this is a good facility on mobile, but the request was releted on the material concept in wich the picker pop up in a full screen dialog and yuo can scroll the calendare swiping up and down. this behaviour on mobile is better and more intuitive. let me know what you think.
